src/popover/popover.js

```diff
@@ -16,4 +16,25 @@ angular.module( 'ui.bootstrap.popover', [ 'ui.bootstrap.tooltip' ] )
 
 .directive( 'popover', [ '$tooltip', function ( $tooltip ) {
   return $tooltip( 'popover', 'popover', 'click' );
+}])
+
+.directive( 'popoverTemplatePopup', [ function () {
+  return {
+    restrict: 'EA',
+    replace: true,
```

Comment:
I'm faithfully following what already is in place:
https://github.com/angular-ui/bootstrap/blob/master/src/popover/popover.js#L11